### ISSUE TITLE
I can't stop the afplay process easily

### ISSUE THREAD

**Issue by joeheyming** [leoliu/play-sound-osx#1]
You have play-sound, but there is no stop-sound

**Comment by leoliu** [comment]
Indeed, this is a tradeoff. The real play-sound-internal is blocking so you can C-g to quit. But I found blocking get in the way so this play-sound-osx used async. I think if you change `start-process` to `call-process` you will get something closer to the primitive and `C-g` to quit/stop.

HTH,
Leo

**Comment by Devon7** [comment]
In my tests, the real _play-sound-internal_ locks up **Emacs** so completely, repeating `C-g` fails to quit.
On the other hand, `C-g C-g C-g` would kill **afplay** but for that bug which runs it asynchronously
unlike the real one which runs synchronously, i.e., Emacs waits for it to finish playing.

Take a look at https://github.com/Devon7/play-sound-osx which attempts to fix that bug.
